src/App.tsx

```diff
@@ -21,11 +39,12 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            onChange={(event) => handleChangePerPage(event)}
           >
-            <option value="3">3</option>
-            <option value="5">5</option>
-            <option value="10">10</option>
-            <option value="20">20</option>
+            <option value={3}>3</option>
+            <option selected value={5}>5</option>
+            <option value={10}>10</option>
+            <option value={20}>20</option>
```

Comment:
```suggestion
            {[3, 5, 10, 20].map(option => (
              <option key={option} value={option}>{option}</option>
            )}
```
consider to write in constant array of possoble option and render option with mapping
something like that